### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by wheresaddie**
minor edits

### CODE REVIEW

**Review by apopiak — APPROVED** [review]
LGTM

**Comment by apopiak** [comment]
This index.md was adjusted from the forkless runtime upgrade tutorial, so we might want to apply some of your suggestions to it as well.

**Comment by danforbes** [comment]
@apopiak can you be more specific about the changes that you think should be applied to the the tutorial? I have opened a PR https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/772 that reverts many of the changes in this one but I'm happy to hear dissenting opinions. My apologies for missing this PR and some of the other related ones. @athei please feel free to comment on #772 as well - I would like to try our best to come to some type of consensus on this issue.

File: docs/tutorials/runtime-migration/index.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ---
-title: Introduction
+Introduction
```
Comment (apopiak) [inline docs/tutorials/runtime-migration/index.md:2]: different from the other tutorial

File: docs/tutorials/runtime-migration/index.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 ---
 
-One of the defining features of the Substrate blockchain development framework is its support for
+One of the defining features of the Substrate blockchain development framework is the support for
 **forkless runtime upgrades**. Forkless upgrades are a means of enhancing a blockchain runtime in a
-way that is supported and protected by the capabilities of the blockchain itself. A blockchain's
+way that supports and protects the capabilities of the blockchain itself. A blockchain's
```
Comment (apopiak) [inline docs/tutorials/runtime-migration/index.md:7]: different from the other tutorial

File: docs/tutorials/runtime-migration/index.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 blockchain can hold and also defines the logic for effecting changes to that state. Substrate makes
 it possible to deploy enhanced runtime capabilities (including _breaking_ changes) without a
 [hard fork](../../knowledgebase/getting-started/glossary#fork). These new capabilities expect the
-blockchain's state and storage schema to be configured in the correct way for operation, though.
-If the expected schema and state are not present a [storage migration](../../knowledgebase/runtime/upgrades.md#storage-migrations) is necessary.
+blockchain state and storage schema to be configured in the correct way for operation.
+If the expected schema and state are not present, a [storage migration](../../knowledgebase/runtime/upgrades.md#storage-migrations) is necessary.
 
-This tutorial will use the Substrate Developer Hub
+This tutorial will use the Substrate
```
Comment (apopiak) [inline docs/tutorials/runtime-migration/index.md:15]: different from the other tutorial

File: docs/tutorials/runtime-migration/index.md [hunk 4]
```diff
@@ -33,8 +33,8 @@ will guide you through the process of setting up your development environment. T
 system for runtime development and guide you through the process of extending the capabilities of a
 FRAME runtime by adding a pallet. The [Perform a Forkless Upgrade](../../tutorials/upgrade-a-chain/) tutorial will introduce runtime upgrades.
 
-If you're an experienced developer and wish to skip those
-tutorials, you can clone the
+If you're a more experienced developer and wish to skip those
+tutorials, you can clone the
```
Comment (apopiak) [inline docs/tutorials/runtime-migration/index.md:37]: different from the other tutorial

**Review by apopiak — COMMENTED** [review]
@danforbes These are the places where Addie made suggestions that would also apply to the other tutorial.

**Pull request merged**